controller/flutter/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,551 @@
+// !$*UTF8*$!
```

Comment:
Isn't this an automatically generated file? If yes, please remove.

Reply:
No, this is the project file. which is needed for the project to build.